lib/chef/client.rb

```diff
@@ -450,7 +450,7 @@ def run
 
         if Chef::Config[:why_run] == true
           # why_run should probably be renamed to why_converge
-          Chef::Log.debug("Not running audits in 'why_run' mode - this mode is used to see potential converge changes")
+          Chef::Log.debug("Not running controls in 'why_run' mode - this mode is used to see potential converge changes")
```

Comment:
The actual text would then read "mode" twice which is a little confusing for people.